src/main/tools/checkstyle/checkstyle-suppressions-override.xml

```diff
@@ -5,6 +5,6 @@
 
 <suppressions>
   <suppress files="mvnw" checks="."/>
-  <suppress files="mvnw.cmd" checks="."/>
-  <suppress files=".mvn/wrapper/MavenWrapperDownloader.java" checks="."/>
+  <suppress files="mvnw\.cmd" checks="."/>
+  <suppress files="\.mvn[\\/]" checks="."/>
```

Comment:
Did you try without escaping the period?

Reply:
Just tried without escaping the periods, but it failed 😦